packages/template-retail-react-app/app/pages/checkout/partials/checkout-header.jsx

```diff
@@ -5,25 +5,33 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import React from 'react'
-import {FormattedMessage} from 'react-intl'
-import {Badge, Box, Button, Flex, Center} from '@chakra-ui/react'
+import PropTypes from 'prop-types'
+import {FormattedMessage, useIntl} from 'react-intl'
+import {Badge, Box, Button, Flex, Center, IconButton, useMultiStyleConfig} from '@chakra-ui/react'
 import useBasket from '../../../commerce-api/hooks/useBasket'
 import Link from '../../../components/link'
 import {BasketIcon, BrandLogo} from '../../../components/icons'
+import {noop} from '../../../utils/utils'
 
-const CheckoutHeader = () => {
+const CheckoutHeader = ({onLogoClick = noop()}) => {
     const basket = useBasket()
+    const intl = useIntl()
+    const styles = useMultiStyleConfig('Header')
 
     return (
         <Box px={[4, 4, 8]} bg="white" borderBottom="1px" borderColor="gray.100">
             <Box maxWidth="container.xxxl" marginLeft="auto" marginRight="auto">
                 <Flex h={{base: '52px', md: '80px'}} align="center" justify="space-between">
-                    <Link href="/" title="Back to homepage">
-                        <BrandLogo
-                            width={{base: '35px', md: '45px'}}
-                            height={{base: '24px', md: '32px'}}
-                        />
-                    </Link>
+                    <IconButton
+                        aria-label={intl.formatMessage({
+                            id: 'header.button.assistive_msg.logo',
```

Comment:
The message has been changed from "Back to hompage" to "Logo". I think the "Logo" take makes sense when we used it to describe the ui that didn't have any action associated to it, but in this case we might want to stick to the original message. 

If you agree, you can make a follow up localization ticket to get the translation for that text if there isn't already one created.

Reply:
+1, assistive text should describe what an element _does_, not what it looks like.

Please also keep this as a link, rather than a button. Buttons make the behavior opaque and lack critical accessibility features associated with links.

Reply:
I reverted the button to be a link and made a small adjustment to link component to work with home ref insteaf. No need to tweak any translations